### PR TITLE
releng: Promote kube-cross:v1.15.7-2

### PR DESCRIPTION
**Pull request by cpanato**
Required for go1.15.7 updates: https://github.com/kubernetes/release/issues/1851

/assign @hasheddan @saschagrunert @xmudrii @puerco 

cc: @kubernetes/release-engineering @ameukam

### CODE REVIEW

**Review by hasheddan — APPROVED** [review]
/lgtm

File: k8s.gcr.io/images/k8s-staging-build-image/images.yaml [hunk 1]
```diff
@@ -231,3 +232,12 @@
...
     "sha256:f9f569e6e1da0fa6ac441b625e599bead8ab10f480ad2a84966cd4a0091e76a8": ["v1.12.17-2"]
+- name: kube-cross-amd64
+  dmap:
+    "sha256:31cf2519710cd3186bbb48d3a0d5f2a4d530e001ddf88577da4775212159d6a8": ["v1.15.7-2"]
+- name: kube-cross-arm64
+  dmap:
+    "sha256:9351f3fdc6ef566064613ab624c10a6c6e17a0a72bf0b13ab4c02a168fb7081e": ["v1.15.7-2"]
+- name: kube-cross-ppc64le
+  dmap:
+    "sha256:33eaad4d236c3f7d1261b04e42cbee83aa538f91176e17ff321407db6b9a2f42": ["v1.15.7-2"]
```
Comment (justaugustus) [inline k8s.gcr.io/images/k8s-staging-build-image/images.yaml:243]:
```suggestion
```

`kube-cross` is a manifest list, so we only need to promote the list.

```console
$ skopeo inspect docker://gcr.io/k8s-staging-build-image/kube-cross:v1.15.7-2 --raw
```

```json
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 3486,
         "digest": "sha256:31cf2519710cd3186bbb48d3a0d5f2a4d530e001ddf88577da4775212159d6a8",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 3270,
         "digest": "sha256:9351f3fdc6ef566064613ab624c10a6c6e17a0a72bf0b13ab4c02a168fb7081e",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 3271,
         "digest": "sha256:33eaad4d236c3f7d1261b04e42cbee83aa538f91176e17ff321407db6b9a2f42",
         "platform": {
            "architecture": "ppc64le",
            "os": "linux"
         }
      }
   ]
}
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
@cpanato -- One nit before this is ready!

File: k8s.gcr.io/images/k8s-staging-build-image/images.yaml (hunk 1 above)
Comment (cpanato) [reply k8s.gcr.io/images/k8s-staging-build-image/images.yaml]: understood, thanks.
PTAL

**Pull request merged**